ci/certification/netgear-rax40.yml

```diff
@@ -98,6 +98,26 @@ MAP-4.4.1_ETH:netgear-rax40:
     - when: manual
       allow_failure: true
 
+MAP-4.2.3_ETH_FH5GL:netgear-rax40:
```

Comment:
These are already there, we just merged a commit that adds all certification tests as jobs. So this commit should be dropped.